tensorboard/examples/plugins/example_raw_scalars/BUILD

```diff
@@ -29,3 +29,12 @@ sh_test(
         "manual",  # https://github.com/tensorflow/tensorboard/issues/2987
     ],
 )
+
+py_test(
+    name = "unit_test",
+    size = "small",
+    srcs = ["test.py"],
```

Comment:
Need to declare a dependency on `util.py` so that changes to `util.py`
actually cause this test to be re-run.